### PR TITLE
chore(azure-eventhub): add warning about connection string source

### PR DESCRIPTION
**Pull request by prousso**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Thanks for the PR! Can you elaborate on what you mean please?

Do you mean that a connection string scoped to a single event hub in a namespace is not working, but a namespace-level connection string does work?

**Comment by prousso** [comment]
The purpose of this PR is to warn about storage connection_string created from Shared Access Signature which have `BlobEndpoint=YYYYYYYYYYYYYYYYY;SharedAccessSignature=sv=XXXXXXXXXXX` format aren't allowed

**Comment by tomkerkhove** [comment]
That seems to be an Azure Storage connection string and not an Azure Event Hubs connection string?

File: content/docs/2.5/scalers/azure-event-hub.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ triggers:
 > 
 > C# Applications that use the current `Azure.Messaging.EventHubs` package, Java applications which use the current `azure-messaging-eventhubs` package or Python applications since `azure-eventhub v5`, have to set the `checkpointStrategy`to `blobMetadata`. All these implementations based on the `EventProcessorClient` which stores checkpoint information as blob metadata.
 
+> 💡 **Connection string:** not compatible with connection string created from Shared Access Signature
```
Comment (arschles) [inline content/docs/2.5/scalers/azure-event-hub.md:42]:
```suggestion
> 💡 The connection string is not compatible with connection string created from a Shared Access Signature
```

**Comment by prousso** [comment]
Yes this is a storage connexion string but `storageConnectionFromEnv` must be connexion string.
My purpose is about SAS connexion string for storage

**Comment by tomkerkhove** [comment]
Sorry but I'm still not sure I get the warning 😅 I'd explicitly add that this is for Azure Storage to make it clear since you have to configure 2 connection strings (storage + event hubs)

I presume this applies for all versions, would you mind updating all 2.x docs please?

**Comment by prousso** [comment]
Done

**Review by tomkerkhove — APPROVED** [review]
Thanks a ton and sorry for the confusion

**Comment by tomkerkhove** [comment]
Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what).

**Comment by prousso** [comment]
Done

**Comment by tomkerkhove** [comment]
Thanks a ton @prousso!

**Pull request merged**